js/letters.js

```diff
@@ -15,11 +15,20 @@ var counter = {
   "y" : 0,  "z" : 0
 }
 
-function countLetters(counter, sample_text){
+function countLetters(counter, sample_text) {
   // FIX ME
-}
+  let textChunk = sample_text.toLowerCase();
+    if (sample_text.length === 0) {
+    return;
+  }
+  let letterInput = textChunk.charAt(0);
+    if ([letterInput]) {
```

Comment:
formatting.

Also the square brackets here are not necessary and actually makes the if statement true at all times. 

You'll also need to figure out if the particular character you're getting is in the counter object or not. Check out the object.hasOwnProperty method for a way to double check if a key exists in an object.